assignments/prototype-refactor.js

```diff
@@ -7,3 +7,101 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+class GameObject {
+    constructor (attrs) {
+    this.createdAt = attrs.createdAt;
+    this.name = attrs.name; 
+    this.dimensions = attrs.dimensions; 
+  }
```

Comment:
The indentation here is looking a little messy. I'd highly recommend looking into a VSCode extension like Prettier or Beautify. 🙂